app/src/cli/commands/clone.ts

```diff
@@ -0,0 +1,72 @@
+import * as QueryString from 'querystring'
+import { URL } from 'url'
+
+import { CommandError } from '../util'
+import { openDesktop } from '../open-desktop'
+import { ICommandModule, mriArgv } from '../load-commands'
+
+interface ICloneArgs extends mriArgv {
+  pr?: number
+  branch?: string
+}
+
+const command: ICommandModule = {
+  command: 'clone <url|slug> [path]',
+  description: 'Clone a repository',
+  args: [
+    {
+      name: 'url|slug',
+      required: true,
+      description: 'The URL to clone, or the GitHub repo slug to clone',
+      type: 'string',
+    },
+    {
+      name: 'path',
+      required: false,
+      description: 'The path to clone to',
+      type: 'string',
+    },
+  ],
+  options: {
+    branch: {
+      type: 'string',
+      aliases: ['b'],
+      description: 'The branch to switch to after cloning',
+    },
+    pr: {
+      type: 'number',
+      aliases: ['p'],
+      description: 'The PR branch to switch to after cloning',
```

Comment:
That's only kinda true... this functionality comes with a lot of caveats that I don't think we wanna expose to users. I don't think we should support this.